feature/staticroute/otg_tests/ipv4_static_routing_test/ipv4_static_routing_test.go

```diff
@@ -0,0 +1,192 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ipv4_static_routing
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/attrs"
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	"github.com/openconfig/ondatra"
+	"github.com/openconfig/ondatra/telemetry"
+	"github.com/openconfig/ygot/ygot"
+)
+
+var (
+	dutPorts = map[string]attrs.Attributes{
+		"port1": {
+			IPv4:    "198.51.100.12",
+			IPv6:    "2001:db8:1::12",
+			IPv4Len: 31,
+			IPv6Len: 127,
+			Desc:    "ATE port 1 to DUT port 1",
+		},
+		"port2": {
+			IPv4:    "198.51.100.22",
+			IPv6:    "2001:db8:1::22",
+			IPv4Len: 31,
+			IPv6Len: 127,
+			Desc:    "DUT port 2 to ATE port 2",
+		},
+		"port3": {
+			IPv4:    "198.51.100.32",
+			IPv6:    "2001:db8:1::32",
+			IPv4Len: 31,
+			IPv6Len: 127,
+			Desc:    "DUT port 3 to ATE port 3",
+		},
+		"port4": {
+			IPv4:    "198.51.100.42",
+			IPv6:    "2001:db8:1::42",
+			IPv4Len: 31,
+			IPv6Len: 127,
+			Desc:    "DUT port 4 to ATE port 4",
+		},
+	}
+
+	atePorts = map[string]attrs.Attributes{
+		"port1": {
+			IPv4:    "198.51.100.13",
+			IPv6:    "2001:db8:1::13",
+			MAC:     "02:1a:c0:00:02:01",
+			IPv4Len: 31,
+			IPv6Len: 127,
+		},
+		"port2": {
+			IPv4:    "198.51.100.23",
+			IPv6:    "2001:db8:1::23",
+			MAC:     "02:1a:c0:00:02:02",
+			IPv4Len: 31,
+			IPv6Len: 127,
+		},
+		"port3": {
+			IPv4:    "198.51.100.33",
+			IPv6:    "2001:db8:1::33",
+			MAC:     "02:1a:c0:00:02:03",
+			IPv4Len: 31,
+			IPv6Len: 127,
+		},
+		"port4": {
+			IPv4:    "198.51.100.43",
+			IPv6:    "2001:db8:1::43",
+			MAC:     "02:1a:c0:00:02:04",
+			IPv4Len: 31,
+			IPv6Len: 127,
+		},
+	}
+)
+
+func TestMain(m *testing.M) {
+	fptest.RunTests(m)
+}
+
+func TestStaticRouteSingleDestinationPort(t *testing.T) {
+
+	// Configure a DUT
+	dut := ondatra.DUT(t, "dut")
+
+	for name, attributes := range dutPorts {
+		pn := dut.Port(t, name).Name()
+		ifCfg := &telemetry.Interface{
+			Name:        ygot.String(pn),
+			Description: ygot.String(attributes.Desc),
+		}
+		ifCfg.GetOrCreateSubinterface(0).GetOrCreateIpv4().Enabled = ygot.Bool(true)
+		ifCfg.GetOrCreateSubinterface(0).
+			GetOrCreateIpv4().
+			GetOrCreateAddress(attributes.IPv4).PrefixLength = ygot.Uint8(attributes.IPv4Len)
+
+		dut.Config().Interface(pn).Update(t, ifCfg)
```

Comment:
Please use the new ygnmi library instead of the old `dut.Config()`:
```go
gnmi.Replace(t, dut, gnmi.OC().Interface(pn).Config(), attributes.NewOCInterface(pn))
```